src/libslic3r/PrintConfig.cpp

```diff
@@ -3324,6 +3324,13 @@ def = this->add("filament_loading_speed", coFloats);
     def->readonly = true;
     def->set_default_value(new ConfigOptionBool(true));
 
+    def = this->add("manual_filament_change", coBool);
+    def->label = L("Manual Filament Change");
+    def->tooltip = L("Perform filament changes by pausing print (must be set in custom g-code) for the manual swap of the filament. "
```

Comment:
Will it be more clear if we chagne the tooltip to following?
"Enabling this option will skip the emission of the tool change command(e.g. T0) and custom Change filament G-code at the beginning of the print. This is useful for those who print multimaterial manually."